test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2149,6 +2149,14 @@ def test_batchnorm1d_noaffine(self):
         x = torch.randn(10, 10, 128)
         self.run_test(model, x)
 
+    def test_batchnorm1d_norunningstats(self):
```

Comment:
Please check locally the test case of `TestONNXRuntime_opset12_IRv4_new_jit_API.test_xxx` as well.